asmcli/asmcli

```diff
@@ -1348,7 +1348,7 @@ install_private_ca() {
   local WORKLOAD_IDENTITY; WORKLOAD_IDENTITY="${WORKLOAD_POOL}:/allAuthenticatedUsers/"
   local CA_LOCATION; CA_LOCATION=$(echo "${CA_NAME}" | cut -f4 -d/)
   local CA_POOL; CA_POOL=$(echo "${CA_NAME}" | cut -f6 -d/)
-  local PROJECT_ID; PROJECT_ID="$(context_get-option "PROJECT_ID")"
+  local PROJECT_ID; PROJECT_ID=$(echo "${CA_NAME}" | cut -f2 -d/)
```

Comment:
This PR changes how to obtain PROJECT_ID. Will this PR impact the integration test and user guide?

Reply:
It will affect integration test - previously, behavior was incorrect. No change to user guide